luminoso_api/upload.py

```diff
@@ -104,6 +105,11 @@ def main():
         help="username (defaults to your username on your computer)")
     parser.add_argument('-p', '--password', default=None,
         help="password (you can leave this out and type it in later)")
+    parser.add_argument('-d', '--date-format', default='iso',
```

Comment:
I went with ISO because I thought the most likely use of this would be to upload Compass messages. I can change the default to `epoch`, but I think that that will be used approximately never, because if you're going to mess around and convert your dates anyways, you'll probably just upload them yourself without resorting to the upload script.